src/main/java/com/openshift/client/cartridge/EmbeddableCartridge.java

```diff
@@ -67,13 +67,11 @@ public boolean equals(Object obj) {
 		IEmbeddableCartridge other = (IEmbeddableCartridge) obj;
 		// shortcut: downloadable cartridges get their name only when
 		// they're deployed thus should equal on url only
-		if (isDownloadable()) {
-			if (other.isDownloadable()) {
-				if (getUrl() == null) {
-					return other.getUrl() == null;
-				}
-				return getUrl().equals(other.getUrl());
-			}
+		if (isDownloadable() && other.isDownloadable()) {
+		    if (getUrl() == null) {
+                return other.getUrl() == null;
+            }
+            return getUrl().equals(other.getUrl());
```

Comment:
can you please correct the identication, hard to grasp without looking closely that closing the outer braces happens right below because the identation is not correct starting at line 72